requirements.prod.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.12
 # by the following command:
 #
-#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements.prod.txt requirements.prod.in
+#    pip-compile --generate-hashes --output-file=requirements.prod.txt requirements.prod.in
```

Comment:
nit: This has changed, which changes the setuptools-related output in this file.

Reply:
huh, not sure why or how this changed, I'll revert it